index.js

```diff
@@ -0,0 +1,32 @@
+// this is the main entry point for your full app
+// it serves your frontend & provides access to your API
+
+const express = require('express');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+
+const api = require('./api');
+
+const app = express();
+app.use(cors());
+app.use(bodyParser.json());
+app.use(express.json())
+
+app.use((req, res, next) => {
+  console.log(req.method + ': ' + req.path);
+  next();
+});
+
+app.use('/', express.static(__dirname + '/client/build/'))
+
+app.get('/', (req, res) => {
+  res.sendFile(__dirname + '/client/build/index.html');
+});
+
+app.use('/api', api);
+
+const appointmentsRouter = require('./api/routes/appointments.js')
+app.use('/api/appointments', appointmentsRouter)
```

Comment:
these two lines should be in `/api/index.js`.  the way you have it set up now, `/api/dev.js` and this file are not the same so you can't actually use `dev.js` to test if it will work in deployment.  line 26 here should be then only line that uses your api.

You should use the appointmentsRouter in `/api/server.js`